swap_meet/vendor.py

```diff
@@ -1,2 +1,63 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
+    def __init__(self, inventory=[]):
+        self.inventory = inventory
+        self.item = Item()
```

Comment:
Curios why you added self.item = Item().
 You do not need it here because we will be passing in an inventory and iterating through that inventory(items) in our methods. 
```suggestion
      
```